kairon/shared/pyscript/callback_pyscript_utils.py

```
@@ -578,6 +580,15 @@ def create_vector_collection(collection_name, model_id: str, user: str, emb_size
         from qdrant_client import QdrantClient
 
         db_url = Utility.environment['vector']['db']
+        try:
+            bot_settings = BotSettings.objects(bot=bot, status=True).get()
+        except DoesNotExist:
+            raise Exception("Bot settings not found")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)